utils/page.ts

```diff
@@ -1,33 +1,21 @@
 import { SegmentNode, admin, main } from '@/types/page';
 
-export const getLocationLog = (location: SegmentNode): SegmentNode[] => {
-  const log: SegmentNode[] = [];
-  let curr = location;
-  while (curr.parent !== null) {
-    log.push(curr);
-    curr = curr.parent;
-  }
-  return log.reverse();
+export const getLocationLog = (location: SegmentNode | null): SegmentNode[] => {
+  if (!(location && location !== main)) return [];
```

Comment:
별거 아니지만,,,
```suggestion
  if (!location || location === main) return [];
```
이렇게 하면 not도 하나 줄고 좀 더 알아보기 편하지 않을까 하는 생각,,,!